### ISSUE TITLE
OpenComputers II compatibility

### ISSUE THREAD

**Issue by sentwayfarer** [TeamPneumatic/pnc-repressurized#1044]
### Describe the feature

Since OpenComputers II recently released, compatibility with the new version of the mod would enhance the mod's compatibility even more, especially with networking.

### Reasons why it should be considered

Stated in the Patchouli documentation, Drone Interfaces work with the original OpenComputers. OpenComputers II, the spiritual successor to OpenComputers released on April 7th, is a more refined version of OpenComputers expanding the Minecraft computer mods even further into success, and should have compatibility with PneumaticCraft: Repressurized sometime soon.

### Additional details

https://www.curseforge.com/minecraft/mc-mods/oc2/files

**Comment by desht** [comment]
Yes, I have looked at the OC2 website and compatibility is under consideration.  But I'm going to need a bit more time to properly understand the OC2 API and how best to integrate it with PNC (and preserving as much common code as possible with CC:Tweaker integration).

In short, yes I'd like it to happen, but can't say when at this point.

**Comment by sentwayfarer** [comment]
> 

Yes, that would be amazing! Take as much time as you need.
